accesscontroltool-bundle/pom.xml

```diff
@@ -190,6 +197,12 @@
               <configuration>
                 <instructions>
                     <Embed-Dependency>el-api,jasper-el</Embed-Dependency>
+                    <!-- Required to support 6.1 and 6.2 -->
+                    <Import-Package>
+                        org.apache.jackrabbit.oak.spi.security.authentication.external;version="[1.0,3)",
```

Comment:
The only way to get it to work for AEM 6.1 and 6.2. The minimal parts of the package it is using should really be fairly stable (in fact it was really annoying to find out that the only reason it did not work was the OSGi version!)